lib/components/Switch.js

```diff
@@ -96,11 +103,11 @@ const Switch = props => {
             {checked ? (
               <CheckedState checked={checked} />
             ) : (
-              <UncheckedState checked={checked} />
-            )}
+                <UncheckedState checked={checked} />
+              )}
           </span>
         </span>
-      </label>
+      </Label>
       {label && <span className="ml-2 switch__label">{label}</span>}
```

Comment:
We need to use `Label` here.

Reply:
@goutham-subramanyam 

Just to be more clear, you meant to replace `span` with `Label`, correct?

Reply:
Yes